oxidation/libparsec/crates/core/src/error.rs

```diff
@@ -74,3 +74,13 @@ impl From<TrustchainError> for RemoteDevicesManagerError {
         Self::InvalidTrustchain { exc }
     }
 }
+
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum LoggedCoreError {
```

Comment:
This is fine for now, but in the long run we should probably have an error enum per function, this in order to have precisely known what error can those high-level interface return (which is a lot simpler for the GUI than to have a function that return a generic error type, then not knowing what error should be handled)